src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/ChartAxisConfiguration.tsx

```diff
@@ -239,6 +239,64 @@ const ChartAxisConfiguration = ({
               />
             )}
           </FormGroup>
+          <FormFieldset
+            id={`${id}-label`}
+            legend="Axis label"
+            legendSize="m"
+            hint="Label your axes"
+          >
+            <FormFieldTextInput
+              id={`${id}-label.value`}
```

Comment:
Don't think it's a good idea to use `.` in ids, not entirely sure they're valid in HTML. Would just use a dash to be safe (same for the other IDs)